CHANGELOG.md

```diff
@@ -29,7 +43,7 @@ https://github.com/openenclave/openenclave/issues/3555 tracks this.
      - Fix TOCTOU vulnerability in NULL terminator checks for ocall in/out string parameters.
      - Count/size properties in deep-copied in/out structs are treated as readonly to prevent the host
```

Comment:
Not part of these changes, but could be cleaned up.
```suggestion
     - Count/size properties in deep-copied in/out structs are treated as read-only to prevent the host
```